neurolib/optimize/evolution/evolution.py

```diff
@@ -315,22 +341,33 @@ def initPypetTrajectory(self, traj, paramInterval, POP_SIZE, NGEN, model):
         # Placeholders for individuals and results that are about to be explored
         traj.f_add_parameter("generation", 0, comment="Current generation")
 
-        traj.f_add_result("scores", [], comment="Score of all individuals for each generation")
-        traj.f_add_result_group("evolution", comment="Contains results for each generation")
+        traj.f_add_result(
+            "scores", [], comment="Score of all individuals for each generation"
+        )
+        traj.f_add_result_group(
+            "evolution", comment="Contains results for each generation"
+        )
         traj.f_add_result_group("outputs", comment="Contains simulation results")
 
         # TODO: save evolution parameters and operators as well, MATE_P, MUTATE_P, etc..
 
         # if a model was given, save its parameters
         # NOTE: Convert model.params to dict() since it is a dotdict() and pypet doesn't like that
         if model is not None:
-            traj.f_add_result("params", dict(model.params), comment="Default parameters")
+            params_dict = dict(model.params)
+            # replace all None with zeros, pypet doesn't like None
+            for key, value in params_dict.items():
+                if value is None:
+                    params_dict[key] = 0
+            traj.f_add_result("params", params_dict, comment="Default parameters")
```

Comment:
This is the part that is new

Reply:
so None seed is saved to pypet as 0 seed. Am I right? What if you actually use 0 seed, that one is also saved as 0 to pypet?

Reply:
That's still an unresolved issue tbh... Was an issue before but I didn't want to spend too much time with it.

Reply:
what if seed specifically is saves as string? `str(<any_integer>) = '<any_integer>'` and `str(None) = 'None'`.. than on loading you can simply:
```python
try:
    seed = int(<saved_in_pypet>)
except ValueError:
    seed = None

```

Reply:
AFAIK `pypet` can do strings...

Reply:
it does, nice!